enzy_htp/_config/__init__.py

```diff
@@ -7,10 +7,31 @@
 Author: Chris Jurich <chris.jurich@vanderbilt.edu>
 Date: 2022-07-12
 """
+from typing import List
 
 from .config import Config
 
 config = Config()
 """
 Singleton object for accessing all configurations.
 """
+
+import os
+from enzy_htp.core import file_system as fs
+from enzy_htp.core import _LOGGER
+
+config_file: str = os.path.expandvars('$HOME/.eh_config')
+
+if fs.has_content(config_file):
+    _LOGGER.info(f"Found config file: {config_file}...")
+    lines: List[str] = fs.lines_from_file(config_file)
+    counter: int = 0
+    for ll in lines:
+        tks = ll.split('#')
+        tk = tks[0].strip()
+        if not tk:
+            continue
+        exec(tk)
```

Comment:
The use of exec is quite dangerous. It also makes the syntax of eh_config out of control. Please mark a TODO for changing it in the future.

Reply:
Agreed; I would re-write it myself so that it is safer but it is somewhat low priority for now. I will update in the future when I have time.

Reply:
I went ahead and changed this so that it is handled by a function in `enzy_htp._config.Config.load_config()` class. Solves the security problem with `exec()` and is generally cleaner.